src/components/reusables/Toggle.tsx

```diff
@@ -38,7 +43,7 @@ export default class Toggle extends React.Component<ToggleOwnProps, ToggleState>
  toggle(e) {
    e.preventDefault();
    let status = !this.state.on;
-   this.props.onToggle(status);
+   this.props.onToggle(status, this.props.name);
```

Comment:
I'm surprised that the qa/prod library toggle doesn't complain because the function you pass in for `onToggle` only has the `status` prop but not the `name` prop. Maybe make this optional in the declaration to be explicit.